pytorch_lightning/setup_tools.py

```diff
@@ -40,10 +40,10 @@ def _load_requirements(path_dir: str, file_name: str = 'requirements.txt', comme
     return reqs
 
 
-def _load_readme_description(path_dir: str, homepage: str = __homepage__, version: str = __version__) -> str:
+def _load_readme_description(path_dir: str, homepage: str, version: str) -> str:
```

Comment:
```suggestion
def _load_readme_description(path_dir: str, homepage: str = "", version: str = "") -> str:
```

Reply:
why?

Reply:
since you had to pass "" in the doctest and you are passing as keyword arguments in `setup.py`